README.md

```diff
@@ -1 +1,138 @@
-# leaderboard
\ No newline at end of file
+# To Do List 📝
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="120"  height="auto"/>
+  <br/>
+
```

Comment:
- Please note the logo on your README file is broken. I'd suggest that you ME file. 😉

Reply:
@rotimiazeez Please what you mean about ME file